packages/components/package.json

```diff
@@ -1,6 +1,6 @@
 {
     "name": "@cura/components",
-    "version": "0.0.19",
+    "version": "0.0.0-development",
```

Comment:
Could you give a link with more details or better explain this?

Reply:
Idk if this explains it well, but that's all I can find
https://www.grzegorowski.com/what-are-npm-dist-tags-how-to-use-them

I'll try to explain my intention from this:

### Why?
We should avoid publishing the packages from our local machines or anywhere else except the `semantic-release` script that run on `master`.
If we do that, we'll start to have some inconsistencies, which can get confusing.
As a good practice, the packages in `master` should always be the same as the published package to npm.

### What does `0-0-0-developement` do?
If someone try to publish the package from his local machine, it will get published to `@cura/components@x.x.x-development` instead of `@cura/components@x.x.x`.

> `developement` in this case doesn't mean anything, it's just a label that separates developement version from the main version, and we could have put there anything.

> `semantic-release` will always publish to `@cura/components@x.x.x` and update the latest version no matter what we put in version in package.json.